examples/iot-nirvana/README.md

```diff
@@ -119,7 +135,7 @@ from the temperature sensors:
      bootstrapping script
 2. Copy the `startup.sh` file in the Google Cloud Storage bucket by running the
    following command in the `/app-engine` folder:
-   `gsutil cp src/main/webapp/startup.sh gs://[BUCKET_NAME]/`
+   `gsutil cp src/main/webapp/startup.sh gs://$PROJECT/`
```

Comment:
Same as before, I would suggest creating a $BUCKET_NAME environment variable and using it here.